SPECS/ca-certificates/certdata.base.txt

```diff
@@ -942,3 +942,331 @@ CKA_TRUST_SERVER_AUTH CK_TRUST CKT_NSS_TRUSTED_DELEGATOR
 CKA_TRUST_EMAIL_PROTECTION CK_TRUST CKT_NSS_MUST_VERIFY_TRUST
 CKA_TRUST_CODE_SIGNING CK_TRUST CKT_NSS_MUST_VERIFY_TRUST
 CKA_TRUST_STEP_UP_APPROVED CK_BBOOL CK_FALSE
+
+#
+# Certificate "Microsoft RSA TLS CA 01"
+#
+# Issuer: CN=Baltimore CyberTrust Root,OU=CyberTrust,O=Baltimore,C=IE
```

Comment:
>CN=Baltimore CyberTrust Root,O [](start = 10, length = 30)

I assume these are not related to the other baltimore certs?

Reply:
I'm not sure what you mean, I'll summarize what this is:

1. This intermediate CA cert **is** signed by the Baltimore root CA. However, we're **not** trusting the root Baltimore CA itself with this change, we're trusting only these 2 new intermediate CAs. These CAs are added to the `ca-certificates-base` package btw.
2. There currently are no Microsoft-owned root CAs we could trust, that's why `ca-certificates-base` consists of a few intermediate CA certs.
3. The Baltimore root CA (along with all certs it has issued) is trusted if you install the Mozilla bundle provided by the `ca-certificates` package but that's not part of the core packages, only `ca-certificates-base` is there.